### ISSUE TITLE
How to Upgrade Ghost page points to old version and missing download page

### ISSUE THREAD

**Issue by tedmiston** [TryGhost/Ghost#7848]
### Issue Summary

The [How to Upgrade Ghost](http://support.ghost.org/how-to-upgrade/) page has grown a little stale.

In the first step "1. Fetching the latest Ghost version", the link to https://ghost.org/download/ is a 404.

Additionally, the instructions state to curl "https://ghost.org/zip/ghost-latest.zip, which always gives you the current version."  However this actually gave me an old version, 0.9.0 IIRC.

### Steps to Reproduce

1. Open How to Upgrade page
2. Click link titled "Ghost.org download page"
3. See 404
4. Run curl command listed under Upgrading > 1. Fetching the latest Ghost version
5. Observe that the downloaded zip is not actually the latest version

Any other info e.g. Why do you consider this to be a bug? What did you expect to happen instead?

I expected the download page to exist and the curl command to pull the latest version.

### Technical details:

* Ghost Version: n/a
* Node Version: n/a
* Browser/OS: Chrome/OS X
* Database: n/a

**Comment by ErisDS** [comment]
Hi @tedmiston thanks for reporting!

I've fixed the /download/ URL to redirect to the new /developers/ page. 

I tried fetching ghost-latest.zip in a number of different ways (browser, curl, etc), and they all gave me 0.11.4. When did you try this? If you access it via the browser, do you still get 0.9?

If you have any more details of how you retrieved an old version of Ghost via the latest zip let me know and I'll look further into it (also if anyone else has experienced this please shout!) However, knowing how it works I'm a bit flummoxed! 

Gonna close this for now as I've fixed the issue I could reproduce and there is not a problem with Ghost itself :)

**Comment by tedmiston** [comment]
Hi @ErisDS, thanks for the quick fix.  I curled just before I made the issue. This could have been a caching issue on my box which was running 0.9.0. Curling it now is returns the 0.11.4 zip. 👍 

By the way, on the Ghost Developers page, it looks like "What's new" points to the blog post for 0.11.3 (https://dev.ghost.org/ghost-0-11-3/) instead of 0.11.4 (https://dev.ghost.org/ghost-0-11-4/).  I looked for a website repo for ghost.org to send a one-line fix PR but I didn't see one listed.
